Lib/test/test_inspect/test_inspect.py

```diff
@@ -1732,6 +1732,72 @@ class C(metaclass=M):
         attrs = [a[0] for a in inspect.getmembers(C)]
         self.assertNotIn('missing', attrs)
 
+    def test_getmembers_function(self):
+        members = inspect.getmembers(mod.eggs)
+        callables = inspect.getmembers(mod.eggs, predicate=callable)
+        self.assertIn(('__name__', 'eggs'), members)
+        self.assertIn(('__doc__', 'A docstring.'), members)
+        self.assertTrue(any(name == '__call__' for name, _ in callables))
+
+    def test_getmembers_code(self):
+        code = mod.eggs.__code__
+        members = inspect.getmembers(code)
+        integers = inspect.getmembers(code, predicate=lambda x: isinstance(x, int))
+        self.assertIn(("co_name", "eggs"), members)
+        self.assertIn(("co_argcount", code.co_argcount), members)
+        self.assertIn(("co_argcount", code.co_argcount), integers)
+
+    def test_getmembers_frame(self):
+        members = inspect.getmembers(mod.currentframe)
+        dicts = inspect.getmembers(
+            mod.currentframe, predicate=lambda x: isinstance(x, dict)
+        )
+        self.assertIn(("f_code", mod.currentframe.f_code), members)
+        self.assertIn(("f_globals", mod.currentframe.f_globals), members)
+        self.assertIn(("f_globals", mod.currentframe.f_globals), dicts)
+
+    def test_getmembers_traceback(self):
+        try:
+            1 / 0
+        except Exception as e:
```

Comment:
Not that it really matters, but we might as well be explicit:
```suggestion
        except ZeroDivisionError as e:
```

Reply:
there is also similar code, maybe it should be fixed too:
https://github.com/python/cpython/blob/d24a22e9b6377797c3b602945347096fbe065670/Lib/test/test_inspect/test_inspect.py#L158-L166